<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OBData</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Escala</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="scroll_Slider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="scrollLabel">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Periodo da Média</string>
          </property>
          <property name="buddy">
           <cstring>timeEdit_2</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit_2">
          <property name="currentSection">
           <enum>QDateTimeEdit::MinuteSection</enum>
          </property>
          <property name="displayFormat">
           <string>mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="okMediaButton">
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Definir origem</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="currentSection">
         <enum>QDateTimeEdit::SecondSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
        <property name="currentSectionIndex">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Velocidade</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="RTPlot" name="Speed_RTP" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Consumo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="RTPlot" name="RPM_RTP" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="RTPlot" name="FuelLevel_RTP" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="RTPlot" name="MAF_RTP" native="true"/>
        </item>
        <item row="1" column="1">
         <widget class="RTPlot" name="ThrottlePos_RTP" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Temperaturas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="RTPlot" name="IntakeTemp_RTP" native="true"/>
          </item>
          <item>
           <widget class="RTPlot" name="CoolantTemp_RTP" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Resumo</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>291</width>
          <height>271</height>
         </rect>
        </property>
        <property name="title">
         <string>Cálculo Total</string>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>181</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Distância Percorrida (km):</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>191</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Admissão de Ar (g):</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="distTotalNumber">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>90</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>10</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QLCDNumber" name="combTotalNumber">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>150</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>10</number>
         </property>
        </widget>
        <widget class="QPushButton" name="calculoTotalButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Calcular</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="rpmTotalNumber">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>210</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>10</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>191</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Rotações do Motor:</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>291</width>
          <height>271</height>
         </rect>
        </property>
        <property name="title">
         <string>Personalizado</string>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>181</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Distância Percorrida (km):</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>191</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Admissão de Ar (g):</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="distPerNumber">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>110</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>10</number>
         </property>
        </widget>
        <widget class="QLCDNumber" name="combPerNumber">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>170</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>10</number>
         </property>
        </widget>
        <widget class="QPushButton" name="calculoPerButton">
         <property name="geometry">
          <rect>
           <x>214</x>
           <y>50</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Calcular</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="calculoInTimeEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>91</width>
           <height>26</height>
          </rect>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::SecondSection</enum>
         </property>
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="calculoFimTimeEdit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>91</width>
           <height>26</height>
          </rect>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::HourSection</enum>
         </property>
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Início</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Fim</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>191</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Rotações do Motor:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="rpmPerNumber">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>230</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>10</number>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Configuração</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Conexão</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="conectar_Button">
             <property name="text">
              <string>Conectar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="desconectar_Button">
             <property name="text">
              <string>Desconectar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>                Status da Conexão:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="conexao_Label">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Protocolo de Comunicação:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="protocolo_Label">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>                  Porta da Conexão:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="porta_Label">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QTextBrowser" name="saida_Browser"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFerramentas">
    <property name="title">
     <string>Comandos</string>
    </property>
    <addaction name="actionIniciar"/>
    <addaction name="actionContinuar"/>
    <addaction name="actionPausar"/>
    <addaction name="actionReiniciar"/>
    <addaction name="actionReescalar"/>
   </widget>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionAjuda"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuFerramentas"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir_log"/>
   <addaction name="actionSalvar_log"/>
   <addaction name="separator"/>
   <addaction name="actionIniciar"/>
   <addaction name="actionContinuar"/>
   <addaction name="actionPausar"/>
   <addaction name="actionReiniciar"/>
   <addaction name="actionReescalar"/>
   <addaction name="separator"/>
   <addaction name="actionAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir_log">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Abrir log</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Carregar um novo log de dados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalvar_log">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Salvar log</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Salvar este log de dados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/run_blue.png</normaloff>:/images/run_blue.png</iconset>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Iniciar a plotagem de dados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionContinuar">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/play_blue.png</normaloff>:/images/play_blue.png</iconset>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Retomar a aquisição de dados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionPausar">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/pause_blue.png</normaloff>:/images/pause_blue.png</iconset>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pausar a aquisição de dados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionReiniciar">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/reload.png</normaloff>:/images/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reinicia a aquisição de dados. Dados não salvos serão perdidos.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionReescalar">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/rescale2.png</normaloff>:/images/rescale2.png</iconset>
   </property>
   <property name="text">
    <string>Reescalar</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reescalar todos os gráficos.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/qt.png</normaloff>:/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Ajuda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RTPlot</class>
   <extends>QWidget</extends>
   <header>rtplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
